sdk/python/packages/flet-core/src/flet_core/dropdown.py

```diff
@@ -38,13 +39,21 @@ def __init__(
         assert key is not None or text is not None, "key or text must be specified"
```

Comment:
This assertion is needs to be updated.
We need to move it into `before_update()`, and check that either `key`, `text`, or `content` is provided.

Also, what's the difference between `text` and `key`? If there isn't, can we get rid of `key` in favour of `text`?

Reply:
Well, this assertion is kind of OK:
- If `content` is not provided then you have to provide either `key` or `text`.
- If `content` is provided you still need to provide `key` or/and `text` that will be used as a key.

Re: `key` and `text`:
- If `key` is not provided then `text` is used as a key.
- If `text` is not provided then `key` is used as a key.
- If both `key` and `text` are provided then `key` is used as a key and `text` as display value.

Reply:
Ah, alright makes sense.
But I think it needs to be moved to `before_update` so we can assert more often, instead of just once at instanciation.